packages/fxa-auth-server/lib/senders/emails/locales/en-US.ftl

```diff
@@ -0,0 +1,7 @@
+sync-reminder = Here's your reminder to sync devices.
+sync-description = It takes two to sync. Syncing another device with Firefox privately keeps your bookmarks, passwords and other Firefox data the same everywhere you use Firefox.
+
+random1 = Or, install on <a %(desktopLinkAttributes)s>another desktop device</a>.
```

Comment:
Still need to dig a little bit more into the localization part, so leaving it here as is for now